api/package.json

```diff
@@ -29,13 +29,18 @@
     "@types/express": "^4.17.2",
     "@types/mongoose": "^5.7.1",
     "@types/morgan": "^1.9.1",
+    "@types/express-session": "^1.17.0",
     "cors": "^2.8.5",
     "dotenv": "^7.0.0",
     "express": "^4.17.1",
+    "express-cas-authentication": "^0.1.0",
+    "express-session": "^1.17.1",
     "mongodb": "^3.2.3",
     "mongoose": "^5.8.1",
     "morgan": "^1.10.0",
-    "typescript": "^3.8.3"
+    "typescript": "^3.8.3",
+    "passport": "^0.4.1",
+    "passport-local": "^1.0.0"
```

Comment:
We can probably yeet these, right? Do we use it for anything?

Reply:
indeed that was going to be another attempt